release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1724,6 +1737,8 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=19485
 
 *Networking*
 
+* Previously, an incorrect Keepalived setting could result in the VIP ending up on an incorrect system, and unable to move back to the correct system. With this update, the incorrect Keepalived setting is removed, so that the VIP ends up on the correct system. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1916890[*BZ#1916890*])
```

Comment:
```suggestion
* Previously, an incorrect Keepalived setting sometimes resulted in the VIP ending up on an incorrect system and unable to move back to the correct system. With this update, the incorrect Keepalived setting is removed so that the VIP ends up on the correct system. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1916890[*BZ#1916890*])
```